keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,474 @@
+Nevertheless, we define this specification because it is an obvious starting point for the long-term, meaningful support of a full-fledged HostProcess feature in Kubernetes.
+
+- Ability to access the APIServer using pod mounted service accounts from a hostProcess pod.
+- Ability to create and manage host level networking (HCN) rules from a Windows hostProcess pod.
+- Ability to launch hostProcess containers which share the IP address of a Windows node.
+- Ability to launch hostProcess containers which can run other privileged Windows system API (to be specified further in the future).
+- Ability for pods to bind to host network interfaces on windows (requires hostProcess pods for scheduling the pod itself).
+- Ability for nodes to continue participating in a K8s cluster after rebooting (requires hostProcess pods for testing).
+
+#### Active Directory
+
+- Ability to run a pod as a GMSA User with only be able to access allowed resources
+- Ability to read and write from local storage using ActiveDirectory credentials
+- Observed lack of Ability to read and write from local storage when ActiveDirectory protected resources credentials are not present
+- The behavior of the `RunAsUserName` field for Windows pods should be that it is supported (i.e. pods can use this), but that there are no guarantees around volume permissions and access when using this field.
+
+#### Network Policies
+
+Note that we do not include UDP, ipv6, in the initial definitions for NetworkPolicy conformance on Windows. Since NetworkPolicy testing support on Windows clusters was just recently added to Kubernetes core, we err on the side of a conservative definition for Windows NetworkPolicy which is sufficient to address the needs of *most* legacy network security applications.
+
+- Ability to protect IPv4 pods from accessing other pods when TCP NetworkPolicies are present that block specific pod connectivity.
+- Ability to protect IPv4 pods from accessing other pods when TCP NetworkPolicies are present that block specific namespace connectability.
+
+#### Windows Advanced Networking and Service Proxying
+
+These apply to Windows Server 2019 and up (we expect windows 2004 to be out of support once this KEP merges).
+
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over NodePort Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over NodePort Endpoints.
+- Ability to run IPv4/IPv6 dual-stack networking (on supported OS Versions).
+
+Note that, ipv4 and ipv6 are both known to be highly CNI (overlay vs non-overlay) and OS (2019 vs 2022) dependent.
+
+#### Windows Worker Configuration
```

Comment:
Is there more than this? things like the variables features enabled, ssh, versions of the kbs installed, windows updates disabled/enabled

Reply:
how are these going to be verified? We don't have any tests that do this today?

Reply:
adding asection about how these arent actually automatically verified. 
I didnt add KBs but i did add RDP/SSH details.
comimting shortly